src/ArchiveAdmin.php

```diff
@@ -282,7 +282,7 @@ public function getOtherModelSelectorField($currentModel = '')
         );
         $modelSelectField->setAttribute(
             'data-others-archive-url',
-            $this->Link('/')
+            rtrim($this->Link('/'), '/')
```

Comment:
Don't do this. Respect trailing slash config.

Reply:
Updated